modular_splurt/code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -0,0 +1,15 @@
+// Function for updating back sprites
+/mob/living/carbon/human/update_inv_back()
+	. = ..()
+	
+	// Check for hidden backpack trait
+	if(HAS_TRAIT(src, TRAIT_HIDE_BACKPACK))
+		..()
```

Comment:
Is it necessary to call `..()` twice in the same proc?

Reply:
This was an accidental inclusion. Further testing revealed it to be unnecessary.